test/sass/mixins/spacing.spec.scss

```diff
@@ -289,8 +289,29 @@
       }
 
       @include expect {
-        padding: 0 48px 128px 256px;
-        padding: 0 3rem 8rem 16rem;
+        html[dir="ltr"] &:not([dir]),
+        &[dir="ltr"] {
+          padding-left: 256px;
+          padding-left: 16rem;
+          padding-right: 48px;
+          padding-right: 3rem;
+        }
+        html[dir="rtl"] &:not([dir]),
+        &[dir="rtl"] {
+          padding-right: 256px;
+          padding-right: 16rem;
+          padding-left: 48px;
+          padding-left: 3rem;
+        }
+        html[dir][dir] & {
+          padding-inline-start: 16rem;
+          padding-inline-end: 3rem;
+          }
+        padding-top: 0;
+        padding-bottom: 128px;
+        padding-bottom: 8rem;
+        padding-block-start: 0;
+        padding-block-end: 8rem;
```

Comment:
Going to see about how to combine `block` and `inline` to reduce the amount of separate fallbacks.

Reply:
Sounds like a separate tidy-up issue, as not directly related to the bug this PR addresses.

Reply:
There are a few other cases it fixes, will open a PR against this PR. 🐢

Reply:
https://github.com/davidcmoulton/libero-pattern-library/pull/1